### ISSUE TITLE
Scrollbars in slice view

### ISSUE THREAD

**Issue by apitts** [airbnb/caravel#634]
I am seeing scrollbars around the slice (e.g. if it's a treemap then there are vertical and horizontal scrollbars) in that slice view. It's just a couple of percent of scroll in each direction but means that you can't see the whole thing in that view and interferes with screenshots . No such scrollbars in the dashboard view which is my current workaround.

**Comment by ascott** [comment]
thanks @apitts. would you mind attaching a screenshot?

**Comment by apitts** [comment]
I'm afraid all my data is confidential at the moment @ascott which is why I couldn't do so initially but I will try to replicate with the test data set when I get a minute.

**Comment by apitts** [comment]
@ascott I decided to just cut out the data...please see below
![export](https://cloud.githubusercontent.com/assets/4076720/16252092/c741b65c-386e-11e6-80eb-088be762956c.png)

**Comment by ascott** [comment]
thanks @piits, super helpful!

**Comment by ascott** [comment]
i wasn't able to repro in chrome, safari or firefox on mac. what browser/os are you using?

![-slice energy sankey](https://cloud.githubusercontent.com/assets/130878/16256795/5ac2b7fc-3809-11e6-8b2e-2ec105e3431b.png)

**Comment by apitts** [comment]
Chrome Version 47.0.2526.106 (64-bit). Ubuntu 14.04.

**Comment by apitts** [comment]
1920 x 1080 screen resolution.

**Comment by ascott** [comment]
i'm on Chrome 51.0.2704.103 (64-bit) on Yosemite. 

could be the way chrome on ubuntu handles svg width/height/padding? or maybe v47 had a bug in it?

**Comment by apitts** [comment]
Thanks. I'll try upgrading my Chrome soon to rule that out.

**Comment by apitts** [comment]
@ascott - Pleased to say I have made some progress on this! Upgraded to Chrome Version 51.0.2704.106 (64-bit). The issue was still there. Wanted to rule out anything in my fork so ran the example data and standard version. Still there. 

Then I used window resizer (https://chrome.google.com/webstore/detail/window-resizer/kkelicaakdanhinjdeammmilcgefonfh?hl=en) to change the window size to 1280 x 1024 and hit refresh on my browser. Scroll bars were gone! But the strangest part is now I can't get them back. Either in my fork or the standard version. So, maybe it was upgrading Chrome. A bit of a mystery. 

I'm going to close this issue though now and if I ever figure it out completely I'll let you know.
